README.md

```diff
@@ -221,18 +250,22 @@ Build a production Docker image for your Express.js application using the follow
 
    ```sh
    docker build --tag nodeserver-run:1.0.0 --file Dockerfile-run .
-   # OR docker build -t nodeserver-run:1.0.0 -f Dockerfile-run .
```

Comment:
I think this just confuses things, most of the command options used in the README have long and short options, the longer are better for teaching, people can use the short forms themselves if they want.